store/sqlite.go

```diff
@@ -143,6 +146,14 @@ func (db *DB) Open() (err error) {
 				" Bytebase create the latest schema. If you are running in production and don't want to reset the data, you can contact support@bytebase.com for help",
 				err)
 		}
+
+		if err := db.pgSeed(verBefore, verAfter); err != nil {
+			return fmt.Errorf("failed to seed: %w."+
```

Comment:
Needs to update this with pg specific info

Reply:
Done